docs/src/modules/reference/pages/release-notes.adoc

```diff
@@ -12,6 +12,10 @@ Current versions
 
 == November 2024
 
+* https://github.com/akka/akka-projection/releases/tag/v1.6.2[Akka Projections 1.6.2]
+
+* https://github.com/akka/akka-persistence-dynamodb/releases/tag/v2.0.1[Akka DynamoDB 2.0.1]
```

Comment:
DynamoDB also announced on the SDK release notes?

Reply:
This is the all akka release feed. We'll see if that works out, but that is what is decided.